### PR TITLE
rg-253: * move header to ResumesHeader

### PR DESCRIPTION
**Pull request by MaxPoplavskiy**
image

### CODE REVIEW

**Comment by MaxPoplavskiy** [comment]
[Move resume header to HeaderWrapper and remove AvatarHeader from resumes pages](https://trello.com/c/NUy17F3l/253-move-resume-header-to-headerwrapper-and-remove-avatarheader-from-resumes-pages)

**Review by Valentyn13 — APPROVED** [review]

**Review by vlad-blip — CHANGES_REQUESTED** [review]
Where did `Share resume button` go?🤔

File: frontend/src/bundles/common/components/header-wrapper/components/resumes-header/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.resume_header__container {
+    display: flex;
+    align-items: center;
+    gap: 24px;
+}
```
Comment (vlad-blip) [inline frontend/src/bundles/common/components/header-wrapper/components/resumes-header/styles.module.scss:5]:
```suggestion
.share__icon {
    width: 25px;
    height: 25px;
    background-color: var(--color-black);
}
```

File: frontend/src/bundles/common/components/header-wrapper/components/resumes-header/resumes-header.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import { useCallback } from 'react';
+import { renderToString } from 'react-dom/server';
+
+import shareIcon from '~/assets/img/share-icon.svg';
```
Comment (ValentinaTenus) [inline frontend/src/bundles/common/components/header-wrapper/components/resumes-header/resumes-header.tsx:4]: consider import svg as react component
```suggestion
import ShareIcon from '~/assets/img/share-icon.svg?react';
```

File: frontend/src/bundles/common/components/header-wrapper/components/resumes-header/resumes-header.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        <Header>
+            <div className={styles.resume_header__container}>
+                <RegularButton
+                    onClick={createResumeAccessLink}
+                    variant={ButtonVariant.GHOST}
+                >
+                    <div
+                        className={styles.share__icon}
+                        style={{ maskImage: `url(${shareIcon})` }}
+                    ></div>
```
Comment (ValentinaTenus) [inline frontend/src/bundles/common/components/header-wrapper/components/resumes-header/resumes-header.tsx:41]:
```suggestion
                    <div className={styles.share__icon}
                        <ShareIcon />
                    </div>
```

Comment (MaxPoplavskiy) [reply frontend/src/bundles/common/components/header-wrapper/components/resumes-header/resumes-header.tsx]: Done

File: frontend/src/bundles/common/components/header-wrapper/components/resumes-header/resumes-header.tsx (hunk 2 above)
Comment (MaxPoplavskiy) [reply frontend/src/bundles/common/components/header-wrapper/components/resumes-header/resumes-header.tsx]: Done

**Review by vlad-blip — APPROVED** [review]

File: frontend/src/bundles/common/components/header-wrapper/components/resumes-header/resumes-header.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import { useCallback } from 'react';
+import { renderToString } from 'react-dom/server';
+
+import ShareIcon from '~/assets/img/share-icon.svg?react';
+import { ButtonVariant } from '~/bundles/common/enums/enums';
+import { useResumes } from '~/bundles/common/hooks/hooks';
+import { ResumeAiReview } from '~/bundles/resume/components/components';
+import { ResumeEditor } from '~/bundles/resume/components/resume-editor/resume-editor';
+
+import { Header, RegularButton } from '../../../components';
```
Comment (Icerock10) [inline frontend/src/bundles/common/components/header-wrapper/components/resumes-header/resumes-header.tsx:10]:
```suggestion
import { Header, RegularButton } from '~';
```

**Pull request merged**